xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -65,6 +65,7 @@ constexpr int DVD_AUDIO_LANG_EXT_DIRECTORSCOMMENTS2 = 4;
 static int dvd_inputstreamnavigator_cb_seek(void * p_stream, uint64_t i_pos);
 static int dvd_inputstreamnavigator_cb_read(void * p_stream, void * buffer, int i_read);
 static int dvd_inputstreamnavigator_cb_readv(void * p_stream, void * p_iovec, int i_blocks);
+static void dvd_logger(void* opaque, dvdnav_logger_level_t level, const char* fmt, va_list va);
```

Comment:
isn‘t the void* typically called `userData` or similar ? 🤔

Reply:
they call it `priv` in the lib and `foo` in vlc :) I called it opaque as it is an opaque data pointer and can be literally anything (it's not used internally). I'll change to priv just to keep consistency with the lib usage